tests/views/test_general_course_data_view.py

```diff
@@ -226,3 +236,21 @@ def test_get_with_course_id_for_other_site(self):
             view = self.view_class.as_view({'get': 'retrieve'})
             response = view(request, pk=str(course.id))
             assert response.status_code == 403
+
+    def test_get_search(self):
```

Comment:
There's a couple of [good reasons for not to write `for`-loops inside tests](https://automationrhapsody.com/data-driven-testing-junit-parameterized-tests/). I won't expand here, but give the alternative:

```suggestion
    @pytest.mark.parametrize('search_term', SEARCH_TERMS)
    def test_get_search(self, search_term):
```